README.md

```diff
@@ -1,10 +1,8 @@
 # napari-btrack
 
-[![License](https://img.shields.io/pypi/l/napari-btrack.svg?color=green)](https://github.com/quantumjot/napari-btrack/raw/master/LICENSE)
-[![PyPI](https://img.shields.io/pypi/v/napari-btrack.svg?color=green)](https://pypi.org/project/napari-btrack)
-[![Python Version](https://img.shields.io/pypi/pyversions/napari-btrack.svg?color=green)](https://python.org)
```

Comment:
Why remove these?

Reply:
just because there's no `napari-btrack` package on PyPI yet, so the badge would display 'package or version not found'

Reply:
Oh yeah, I forgot, it was `napari-btrack-reader` that was on there